src/System.Drawing.Common/tests/Printing/MarginsTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
@akoeplinger Yes, this is special guidance we've received from our lawyers, specifically when using mono code.